components/Input/input.css

```diff
@@ -0,0 +1,26 @@
+.input {
+  display: inline-block;
+  vertical-align: middle;
+  border: 1px solid transparent;
+  padding: 5px 10px;
+  font-size: 1rem;
+  border-radius: 2rem;
+}
+.input:focus {
+  outline: none;
```

Comment:
outline none  is not considered good practice, instead use this  outline: 2px solid transparent; outline-offset: 2px;

Reply:
sure, I'll make suggested change